docs/aerodromes/classd/Parafield.md

```diff
@@ -33,32 +50,42 @@ Circuits to be flown at `A010`
 | 26L  | Left   | -     |
 | 26R  | Right  | -     |
 
-## VFR Inbound Procedures
+## VFR Operations
+### Arrivals
 | VFR Approach Point | RWYs 03 | RWYs 21 | RWYs 08 | RWYs 26 |
 | ----------------| --------- | ---------- | ----- | ----- |
 | OHB   | Join base (or downwind) RWY 21R | Join base (or downwind) RWY 03L | Join final RWY 08L| Join downwind RWY 26R | 
 | DMW | Join downwind RWY 03R | Join 3NM final RWY 21L| Join downwind RWY 08L | Join final RWY 26R |
 
-!!! note
-    Visually monitor all aircraft to ensure tracking South of Bolivar Strobe, to ensure aircraft remain outside of Edinburgh CTR. 
+!!! warning "Important"
+    Visually monitor all aircraft to ensure tracking South of **Bolivar Strobe** (located at the Southern tip of the EDN CTR), to ensure aircraft remain outside of EDN CTR. 
     <figure markdown>
-    ![Bolivar](img/bolivar.png){ width="700" }
+    ![Bolivar](img/bolivar.png){ width="400" }
     </figure>
 
+## IFR Operations
+### Departures
+IFR Departures from YPPF shall be assigned **PF RADAR** SID when Runway 03/21 is in use.
+
 ## ATIS
-### Operational Info
-ATIS broadcast **BY DAY** shall follow the format:   
-`RWY 21R (OR 03L) FOR ARRS & DEPS WEST, RWY 21L (OR 03R) ARRS & DEPS EAST`  
-OR   
-`RWY 26R (OR 08L) FOR ARRS & DEPS, RWY 26L (OR 08R) FOR CCT TRAINING`
-   
-ATIS broadcast **BY NIGHT** shall follow the format:  
-`RWY 21R (OR 03L) FOR ALL OPERATIONS`  
+### Runway Nomination
+
+| Priority | Runway Mode |
+| ------ | ------ |
+| =1     | 03L & 03R |
+| =1     | 21L & 21R |
+| =2     | 08L & 08R |
+| =2     | 26L & 26R |
+| 3     | Any Single Runway Operations |
+
+When 2 ADC controllers are online, the ATIS shall be formatted: `RWY 03R/08R/21L/26L FOR ARRS AND DEPS EAST, FREQ 118.7. RWY 03L/08L/21R/26R FOR ARRS AND DEPS WEST, FREQ 124.6`
```

Comment:
Swap frequencies please. 

East/South 124.600
West/North 118.700

Not sure what you want to call them either, as ADC2 (West/North) should be the prime position (CTAF freq)... not familiar with the naming conventions available but PF_E_TWR could work...

Reply:
@slep thoughts?

Reply:
> Swap frequencies please.
> 
> East/South 124.600 West/North 118.700
> 
> Not sure what you want to call them either, as ADC2 (West/North) should be the prime position (CTAF freq)... not familiar with the naming conventions available but PF_E_TWR could work...

@Slep-wt thoughts?

Reply:
> > Swap frequencies please.
> > East/South 124.600 West/North 118.700
> > Not sure what you want to call them either, as ADC2 (West/North) should be the prime position (CTAF freq)... not familiar with the naming conventions available but PF_E_TWR could work...
> 
> @Slep-wt thoughts?

As per [here](https://github.com/vatpac-technology/sops/pull/414#discussion_r1899474126), I'd prefer not to change the position identifer. In terms of swapping ADC1/ADC2 internally that is pretty easy to do if thats how it runs rw.

@LeGiraffe

Reply:
> Leaving as is, Night Ops will be performed on the incorrect frequency, with the opportunity to be "undercut" when the secondary tower logs on and steals majority of meaningful traffic. The lack of ability to combine frequencies across multiple tower positions also hinders this.
> 
> Another issue is (RW) PF ADC1 is going to be called PF-2 (ADC2) for us.
> 
> Any reason for the lack of flexibility with the identifiers, when we're using other letter for other aerodromes already (including TW-S in this PR)?
> 
> PF-E is not perfect, just feel as though there's a better solution out there than PF-2. For example, BK-2 could be BK-C given all it does is CCT's and (almost) nothing else, helping all other units know who is doing what and who to talk/transfer to without reading the book.
> 
> @Slep-wt @alphadelta332
> 
> I need to also check that our other 2's are actually ADC2 RW...

Swapping ADC1 and ADC2's assigned frequencies isnt too difficult to do, we can resolve that quite easily.

In regard to Tamworth, it was done that way because it has an explicit definition for its area of responsibility in the ERSA (Its very hard to get confused as TW ADCS is only responsible for the southern runway during PROPS). 

Again, adding extra information such as explicit responsibility for runways in the controller info may be more appropriate and useful to pilots, as it is simpler to do in terms of the amount of things to edit.

Reply:
The same logic could be applied to BK. While it may not be defined, BK ADC2 is solely responsible for the southern runway during PROPS. Does that mean it too gets BK-S?

My main sticking point is the question; Are we happy calling ADC1 (RW) ADC2 (Here) and vice versa?

I don't feel this one shoe fits all is the most optimal solution for clarity (for both Pilots & controllers).

Reply:
For what it's worth, I agree, and would prefer a cardinal direction designation over a 1/2 designation, where the cardinal direction is consistent and obvious. But ultimately its up to @Slep-wt